pandas/core/generic.py

```diff
@@ -7687,17 +7689,35 @@ def resample(
             By default the input representation is retained.
         loffset : timedelta, default None
             Adjust the resampled time labels.
+
+            .. deprecated:: 1.1.0
+                You should add the loffset to the `df.index` after the resample.
+                like this:
+                ``df.index = df.index.to_timestamp() + to_offset(loffset)``
```

Comment:
you can just say see below, no need to add the example here

Reply:
Fixed in the last commit.